### ISSUE TITLE
result is black

### ISSUE THREAD

**Issue by ayutools** [irasin/Structure-emphasized-Multimodal-Style-Transfer#2]
(PY388) G:\proj\YUs_Ai\02PYTORCH\00style_transfer\HD\00Structure-emphasized-Multimodal-Style-Transfer-master\SEMST_Original>python test.py -c ../content/avril.jpg -s ../style/horse.jpg
CUDA available: GeForce RTX 3090
SEMST_Original_model_state.pth loaded
result saved into files starting with avril_horse

————————————————————————————————————
All is good, but the result is black
win10
pytorch 1.8.1
python 3.8
rtx 3090
cuda 11.1
![avril_horse_style_transfer_demo](https://user-images.githubusercontent.com/22383994/116654130-220a2200-a9bb-11eb-8cfb-022e25291283.jpg)

**Comment by irasin** [comment]
Does any error occur?

I have not tested on the newest Pytorch yet. Show me the log if possible.

**Comment by ayutools** [comment]
No. I tested it on the cpu and everything is good.Where is the log file?

Sin ***@***.***> 于2021年5月1日周六 下午2:00写道：

**Comment by irasin** [comment]
Hi, @vbqgyy 

I tested it on PyTorch1.8 cpu and it seems OK. Check your env again please.

![avril_horse_style_transfer_demo](https://user-images.githubusercontent.com/25549893/117116890-7a795f00-adc1-11eb-8100-57994c3f1946.jpg)

![image](https://user-images.githubusercontent.com/25549893/117117554-3fc3f680-adc2-11eb-92e6-bfad062e317b.png)

**Comment by ayutools** [comment]
There is no problem with cpu, only gpu has a problem。

**Comment by irasin** [comment]
@vbqgyy 

Sorry, I will check it later

**Comment by ayutools** [comment]
That would be great. Thank you.

**Comment by irasin** [comment]
Hi, @vbqgyy 

I have tested on cuda11 + Pytorch1.8,1 and the result is wrong indeed. And the reason why the result is black, is because the output of Model is Nan. 

However, when I tested it on cuda10.1 + Pytorch1.7.1, the result is OK again.
 
I guess there are some version compatibility problems here. I will fix it if I have time. Or you can check it by yourself. Best wish.

**Comment by ayutools** [comment]
@irasin 
Thank you for taking the time to do the test.I am just a beginner.I guess it will take a lot of time and luck to complete this thing.
